python/mqt/core/plugins/qiskit/backend.py

```
@@ -337,6 +256,104 @@ def _build_target(self) -> Target:
 
         return target
 
+    def _add_operation_to_target(self, target: Target, op: fomac.Device.Operation, seen_gate_names: set[str]) -> None:
```

Ruff (0.15.20)

[warning] Too many return statements (7 > 6)

(PLR0911)